client/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:12.18.3-buster-slim
```

Comment:
This Dockerfile is needed to run the node development server when developing. It essentially replicates what we were doing previously to run the app.